### ISSUE TITLE
[GUI] [Err] [RenderEngineManager.cc:461]Failed to load plugin [ogre2] : couldn't find shared library.

### ISSUE THREAD

**Issue by krixdina** [gazebosim/gz-sim#2110]
## Environment
* OS Version: Ubuntu 22.04 
* build from source ,Fortress

I just want to spawn a empty world to get familiar with Gazebo,but a weird problem coming up
![截图 2023-09-01 20-16-10](https://github.com/gazebosim/gz-sim/assets/126226773/b34a131d-508c-42ae-be09-07c41ff16c93)

so ,  what to do to solve this problem

**Comment by iche033** [comment]
do you have `libogre-next-dev` installed? For fotress, check that the version ogre should be 2.2.5. If you don't, install the package and do a clean rebuild of ign-rendering

**Comment by kristinawzy** [comment]
> 

yes , I'd checked out the install and version of package `libogre-next-dev` , and do a clean rebuild of ign-rendering ,but nothing different from the proceding error
![image](https://github.com/gazebosim/gz-sim/assets/126226773/c50e19c6-5ef6-43ac-b244-8420aa048d03)

**Comment by kristinawzy** [comment]
> do you have `libogre-next-dev` installed? For fotress, check that the version ogre should be 2.2.5. If you don't, install the package and do a clean rebuild of ign-rendering

yes , I'd checked out the install and version of package libogre-next-dev , and do a clean rebuild of ign-rendering ,but nothing different from the proceding error
![截图 2023-09-02 15-54-39](https://github.com/gazebosim/gz-sim/assets/126226773/e466f9db-f0fe-4b74-9e12-b5526274ca60)

**Comment by kristinawzy** [comment]
> do you have `libogre-next-dev` installed? For fotress, check that the version ogre should be 2.2.5. If you don't, install the package and do a clean rebuild of ign-rendering

yes , I'd checked out the install and version of package `libogre-next-dev` , and do a clean rebuild of ign-rendering ,but  also something weird happened different from the proceding error
![Screenshot from 2023-09-02 16-31-09](https://github.com/gazebosim/gz-sim/assets/126226773/6e6dadd9-8da1-4582-8878-2c7d6bbeb141)

when try to deal with this problem , I switch to this path `/home/kristina/gz_ws/install/lib/ign-rendering-6/engine-plugins
` there obviously lying the plugin `libignition-rendering-ogre2.so`,why can't load this plugin
![image](https://github.com/gazebosim/gz-sim/assets/126226773/29223cf6-57f4-4ed8-88e8-5d2a9a39675e)

**Comment by iche033** [comment]
looks like the problem is with the `libOgreHlmsPbs_d.so.2.2.7`. The `_d` suffix suggests that it's looking for the debug version of the library and it probably does not exist on your system. It's also looking for version `2.2.7` instead of `2.2.5` (`2.2.5` is the version in our package repo). I think you may have installed ogre from source in debug mode at point in time? You may have to remove other versions of ogre installed in your system

**Comment by wojtess** [comment]
Is there any solution for that type of problem? I did following:
1. run `gz sim world.sdf`
2. save client configuration
3. open new console
4. run `gz gui` in new console
5. load saved client configuration

I got this error:
```
rosuser@wojtess-thinkpad-laptop:~/ros_ws$ gz gui
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-rosuser'
MESA: error: Failed to query drm device.
libGL error: glx: failed to create dri3 screen
libGL error: failed to load driver: iris
[GUI] [Err] [WorldStats.cc:132] Must specify a <topic> to subscribe to world statistics, or set the MainWindow's [worldNames] property.
[GUI] [Err] [Application.cc:480] Failed to load plugin [EntityContextMenuPlugin] : couldn't find shared library.
[GUI] [Err] [Application.cc:480] Failed to load plugin [GzSceneManager] : couldn't find shared library.
[GUI] [Err] [Application.cc:480] Failed to load plugin [SelectEntities] : couldn't find shared library.
[GUI] [Err] [Application.cc:480] Failed to load plugin [Spawn] : couldn't find shared library.
[GUI] [Err] [Application.cc:480] Failed to load plugin [VisualizationCapabilities] : couldn't find shared library.
[GUI] [Err] [WorldControl.cc:136] Must specify a <service> for world control requests, or set the MainWindow's [worldNames] property.
[GUI] [Err] [WorldStats.cc:132] Must specify a <topic> to subscribe to world statistics, or set the MainWindow's [worldNames] property.
[GUI] [Err] [Application.cc:480] Failed to load plugin [Shapes] : couldn't find shared library.
[GUI] [Err] [Application.cc:480] Failed to load plugin [Lights] : couldn't find shared library.
[GUI] [Err] [Application.cc:480] Failed to load plugin [TransformControl] : couldn't find shared library.
[GUI] [Err] [Application.cc:480] Failed to load plugin [CopyPaste] : couldn't find shared library.
[GUI] [Err] [Application.cc:480] Failed to load plugin [ComponentInspector] : couldn't find shared library.
[GUI] [Err] [Application.cc:480] Failed to load plugin [EntityTree] : couldn't find shared library.
```

I am running on docker, can share Dockerfile and running script
